docs/data/date-pickers/base-concepts/base-concepts.md

```diff
@@ -81,7 +81,7 @@ Each _Picker_ is available in a responsive, desktop and mobile variant:
 
 There are many components available, each fitting specific use cases. Use the form below to find the component you need:
 
-{{"demo": "ComponentExplorerNoSnap.js", "hideToolbar": true}}
+{{"demo": "ComponentExplorerNoSnap.js", "hideToolbar": true, "bg": "inline"}}
```

Comment:
I assume it was forgotten, the other demos have it.

Reply:
I'm not so sure. The only other demos without a border are the `basic` demos showing small examples between paragraphs. This is possibly a huge demo, which might not even fit vertically to a screen. I feel that having the border ties the demo together and is more in line with most of the other examples.

Reply:
```suggestion
{{"demo": "ComponentExplorerNoSnap.js", "hideToolbar": true}}
```